src/components/Dashboard/DashboardBannerRow.tsx

```diff
@@ -21,8 +29,20 @@ export const DashboardBannerRow: React.FC<Props> = ({ meet }) => {
     return () => clearInterval(interval);
   }, [meet]);
 
-  return (
-    <div className="w-full shrink-0 h-full lg:shrink flex flex-col items-center">
+  return venueLocation ? (
```

Comment:
i would put the ternary in the link's `to` field because you're duplicating the child jsx in both cases